app/app/controllers/cbv/synchronizations_controller.rb

```diff
@@ -11,7 +11,11 @@ def update
   private
 
   def redirect_if_sync_finished
-    if @pinwheel_account && @pinwheel_account.has_fully_synced?
+    if @pinwheel_account.nil?
+      redirect_to cbv_flow_synchronization_failures_path and return
```

Comment:
`and return` is a terrible rubyism that I'd like us to avoid.

Reason being:
1. I would like it to be very obvious when methods are returning
2. This is hard to see because it's at the end of a line
3. It also relies on the truthiness of `redirect_to` in order to return. That's weird!
4. It uses the `and` keyword instead of the `&&` operator. Although this is [the most controversial ruby style preference of all time](https://github.com/rubocop/ruby-style-guide#and-or-flow), it is undeniable that there can be weird bugs related to the low precedence of `and` and `or`. Best to just avoid them.

tl;dr: no reason not to just:
```suggestion
      return redirect_to cbv_flow_synchronization_failures_path
```